CP04OSSM/CP04OSSM_R00002_ingest.csv

```diff
@@ -38,6 +38,8 @@ mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/w
 mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl11/MOPAK_sn/*.mopak.log,CP04OSSM-SBD11-01-MOPAK0000,recovered_host,Available,
 mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl11/Hydrogen_sn/*.hyd1.log,CP04OSSM-SBD11-02-HYDGN0000,recovered_host,Available,
 mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl11/METBK-Log_sn/*.metbk.log,CP04OSSM-SBD11-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.metbk_ct.dcl.metbk_ct_dcl_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl11/METBK-Log_sn/*.metbk.log,CP04OSSM-SBD11-06-METBKA001,recovered_host,Available,
+#mi.dataset.driver.metbk_ct.metbk_ct_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl11/METBK-CT_sn37-11093/*.hex,CP04OSSM-SBD11-06-METBKA001,recovered_inst,Not Available,No recovered data

```

Comment:
same comments as with early ISSM.  There is an instruments folder on raw data repo but nothing in dcl11, so I think mapping is ok for these?